EIPS/eip-6808.md

```diff
@@ -0,0 +1,680 @@
+
+Limitless Scalability: Centralized solutions require the creation of an account and their availability may be restricted based on location. **KBTs**[^1] do not face regional restrictions or account creation. Decentralized security solutions such as hardware options face scalability issues requiring transport logistics, secure shipping, and vendor. **KBTs**[^1] can be used anywhere around the world by anyone who so wishes, provided they have access to the internet.
+
+Undeniable Anonymity: Frequently, centralized solutions ask for personal information that is stored and subject to prying eyes. Purchasing decentralized hardware solutions are susceptible to the same issues e.g. a shipping address, payment information or a camera recording during a physical cash pick-up. This may be considered by some as infringing on their privacy and asset anonymity. **KBTs**[^1] ensures user confidentially as everything can be done remotely under a pseudonym on the blockchain.
+
+Environmentally Friendly: Since the security features are coded into the standard, there is no need for centralized servers, shipping, or the production of physical object/s. Thus leading to a minimal carbon footprint by the use of **KBTs**[^1], working hand in hand with Ethereum’s change to a _PoS_[^28] network.
+
+Low-Cost Security: The cost of using **KBTs**[^1] security features correlate to on-chain fees, the current **GWEI**[^17] at the given time. As a standalone solution, they are a viable cost-effective security measure feasible to the majority of the population.
+
+Unmatched Security: By calling the `addBinding`[^2] function a **Key Wallet**[^3] is now required for the `allowTransfer`[^4] or `allowApproval`[^5] function. The `allowTransfer`[^4] function requires 4 parameters, `_amount`[^14], `_time`[^15], `_address`[^16], and `_allFunds`[^29], where as the `allowApproval`[^5] function there are 2 parameters, `_time`[^30] and `_numberOfTransfers`[^31]. In addition to this, **KBTs[^1]** have a `safeFallback`[^12] and `resetBindings`[^13] function. The combination of all these prevent and virtually cover every single point of failure that is present with a traditional _ERC-20_[^9], when properly used.
+
+Increased Confidence: With **KBTs**[^1], holders can be confident that their tokens are safe and secure, even if the **Holding Wallet**[^11] or one of the **Key Wallets**[^3] has been compromised.
+
+User Experience: **KBTs**[^1] optional security features improve the overall user experience and Ethereum ecosystem by ensuring a safety net for those who decide to use it. Those that do not use the security features are not hindered in any way. This safety net can increase global adoption as people can remain confident in the security of their assets, even in the scenario of a compromised wallet.
```

Comment:
A lot of this reads more like motivation, instead of rationale. Although the two sections seem similar, the motivation section should justify the existence of the EIP as a whole, while the rationale should explain individual technical decisions made within the EIP.

For example, you might want to explain why `allowTransfer` and `allowApproval` return bools instead of reverting on failure. I assume it's for consistence with ERC-20, but it could be worth noting.